### ISSUE TITLE
Unused code is bundled when using Angular

### ISSUE THREAD

**Issue by ErikNijland** [optimizely/javascript-sdk#686]
## What I wanted to do
I want to include @optimizely/optimizely-sdk to an existing Angular 12.1.1 project.

## What I expected to happen
- No errors in the terminal when running the app with `ng serve`
- No errors in the terminal when building the app with `ng build`
- No errors in the browser console.

## What actually happened
In all three scenarios the following error appears:

```
Warning: /my/local/path/node_modules/@optimizely/optimizely-sdk/lib/shared_types.ts is part of the TypeScript compilation but it's unused.
Add only entry points to the 'files' or 'include' properties in your tsconfig.
```

## Steps to reproduce
Link to repository that can reproduce the issue: https://github.com/ErikNijland/optimizely-angular-cli-bug

- Install the NPM dependencies with `npm ci`
- Run `ng build` or `ng serve`

**`@optimizely/optimizely-sdk` version: 4.6.0** 

**Browser and version: Chrome 91**

**`node` version: v14.17.1** 

**`npm` version: 7.19.1**

**Comment by wpannell** [comment]
We have the same issue.  Is there an update?

**Comment by tonjohn** [comment]
How's this coming along?

**Comment by JamesIves** [comment]
In addition Angular also throws this warning:

```
filename.ts depends on '@optimizely/optimizely-sdk'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies
```

**Comment by mikechu-optimizely** [comment]
For the v5 release we're looking / researching whether we need to ship the CJS and UMDs any more.  We'll also be sliming down by removing the /lib/ dir. 

Relates to #352 and #638

**Comment by raju-opti** [comment]
This issue is fixed in versions >= 4.9.2. Closing this issue
